blob/music.py

```diff
@@ -24,21 +24,41 @@ class Score:
     theme: Theme = Theme.NORMAL
     language: Language = Generic
     tempo: float = 1.0
+    tracks: str = "AUTO"
+    phoneme_fill_in: Phoneme = Phoneme.SIL
```

Comment:
See above.

```suggestion
    tracks: Tuple[int] = (0, 0, 0, 0)
    fill: Phoneme = Phoneme.SIL
```

Don't forget to import the `Tuple` class from `typing` if you decide to apply my suggestion:

```python
from typing import Tuple
```